content/developer/reference/extract_api/implementation.py

```diff
@@ -7,95 +7,119 @@
-print("\nResult saved in results.txt")
-
-if response['result'][document_id]['status_code'] != SUCCESS:
-    print(response['result'][document_id]['status_msg'])  # if it isn't a success, print the error message
-    sys.exit(1)
-
-document_results = response['result'][document_id]['results'][0]
-print("\nTotal:", document_results['total']['selected_value']['content'])
-print("Subtotal:", document_results['subtotal']['selected_value']['content'])
-print("Invoice id:", document_results['invoice_id']['selected_value']['content'])
-print("Date:", document_results['date']['selected_value']['content'])
-print("...\n")
-
-params = {
-    'document_id': document_id,
-    'values': {
-        'total': {'content': 100.0},
-        'subtotal': {'content': 100.0},
-        'global_taxes': {'content': []},
-        'global_taxes_amount': {'content': 0.0},
-        'date': {'content': '2020-09-25'},
-        'due_date': {'content': '2020-09-25'},
-        'invoice_id': {'content': document_results['invoice_id']['selected_value']['content']},
-        'partner': {'content': 'twinnta'},
-        'VAT_Number': {'content': 'BE23252248420'},
-        'currency': {'content': 'USD'},
-        'merged_lines': False,
-        'invoice_lines': {'lines': [{'description': 'Total TVA ',
-                                     'quantity': 1.0,
-                                     'unit_price': 100.0,
-                                     'product': False,
-                                     'taxes_amount': 0.0,
-                                     'taxes': [],
-                                     'subtotal': 100.0,
-                                     'total': 100.0}]
+def validate_results(document_uuid):
+    # This is an example of how to validate the results of the parsing
+    # These values should be the correct values for the document reviewed by the user
+    params = {
+        'document_id': document_uuid,
+        'values': {
+            'total': {'content': 100.0},
+            'subtotal': {'content': 100.0},
+            'global_taxes': {'content': []},
+            'global_taxes_amount': {'content': 0.0},
+            'date': {'content': '2020-09-25'},
+            'due_date': {'content': '2020-09-25'},
+            'invoice_id': {'content': 'INVOICE-0001'},
+            'partner': {'content': 'twinnta'},
+            'VAT_Number': {'content': 'BE23252248420'},
+            'currency': {'content': 'USD'},
+            'merged_lines': False,
+            'invoice_lines': {
+                'lines': [
+                    {
+                        'description': 'Total TVA ',
+                        'quantity': 1.0,
+                        'unit_price': 100.0,
+                        'product': False,
+                        'taxes_amount': 0.0,
+                        'taxes': [],
+                        'subtotal': 100.0,
+                        'total': 100.0
+                    },
+                ]
+            }
         }
     }
```

Comment:
I think we shouldn't provide a pre-filled `values` dictionary, most users won't bother editing it with the correct values.
Providing a skeleton that they can fill in themselves if they want to try out the validation would be better.

Reply:
Rewritten with type hints and ellipsis.